cdap-docs/included-applications/source/etl/plugins/transforms/script.rst

```diff
@@ -43,6 +43,10 @@ will scale the ``'count'`` field by 1024.
 **schema:** The schema of output objects. If no schema is given, it is assumed that the output
 schema is the same as the input schema.
 
+**lookup:** The configuration for the lookup tables to use. For example, if lookup table "purchases" is configured,
+then you will be able to perform operations on that lookup table in your script:
```

Comment:
I'm not sure what to change here, in that I don't know the parameters...is there one lookup table, or any number? looks like the latter. So, perhaps this should be:
```
**lookup:** The configuration of any lookup tables being used. For example, if a lookup table "purchases" 
is configured, then you will be able to perform operations with that lookup table in your script:
```
I'm assuming "with" is more accurate than "on", in that you are not writing, only reading from the lookup table?